codegen-v2/src/codegen/swift/templates/WalletCore.h

```diff
@@ -0,0 +1,80 @@
+// Copyright © 2017-2023 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#import <Foundation/Foundation.h>
+
+//! Project version number for TrustWalletCore.
+FOUNDATION_EXPORT double WalletCoreVersionNumber;
+
+//! Project version string for TrustWalletCore.
+FOUNDATION_EXPORT const unsigned char WalletCoreVersionString[];
+
+#include "TWBase.h"
+#include "TWData.h"
+#include "TWString.h"
+
+#include "TWAnySigner.h"
+
+#include "TWAES.h"
+#include "TWAESPaddingMode.h"
+#include "TWAccount.h"
+#include "TWAnyAddress.h"
+#include "TWBase32.h"
+#include "TWBase58.h"
+#include "TWBase64.h"
+#include "TWBitcoinAddress.h"
+#include "TWBitcoinMessageSigner.h"
+#include "TWBitcoinScript.h"
+#include "TWBitcoinSigHashType.h"
+#include "TWBlockchain.h"
+#include "TWCardano.h"
+#include "TWCoinType.h"
+#include "TWCoinTypeConfiguration.h"
+#include "TWCurve.h"
+#include "TWDataVector.h"
+#include "TWDerivation.h"
+#include "TWDerivationPath.h"
+#include "TWDerivationPathIndex.h"
+#include "TWEthereum.h"
+#include "TWEthereumAbi.h"
+#include "TWEthereumAbiFunction.h"
+#include "TWEthereumAbiValue.h"
+#include "TWEthereumChainID.h"
+#include "TWEthereumMessageSigner.h"
+#include "TWFIOAccount.h"
+#include "TWFilecoinAddressConverter.h"
+#include "TWFilecoinAddressType.h"
+#include "TWGroestlcoinAddress.h"
+#include "TWHDVersion.h"
+#include "TWHDWallet.h"
+#include "TWHRP.h"
+#include "TWHash.h"
+#include "TWLiquidStaking.h"
+#include "TWMnemonic.h"
+#include "TWNEARAccount.h"
+#include "TWNervosAddress.h"
+#include "TWPBKDF2.h"
+#include "TWPrivateKey.h"
+#include "TWPrivateKeyType.h"
+#include "TWPublicKey.h"
+#include "TWPublicKeyType.h"
+#include "TWPurpose.h"
+#include "TWRippleXAddress.h"
+#include "TWSS58AddressType.h"
+#include "TWSegwitAddress.h"
+#include "TWSolanaAddress.h"
+#include "TWStarkExMessageSigner.h"
+#include "TWStarkWare.h"
+#include "TWStellarMemoType.h"
+#include "TWStellarPassphrase.h"
+#include "TWStellarVersionByte.h"
+#include "TWStoredKey.h"
+#include "TWStoredKeyEncryption.h"
+#include "TWStoredKeyEncryptionLevel.h"
+#include "TWTHORChainSwap.h"
+#include "TWTezosMessageSigner.h"
+#include "TWTransactionCompiler.h"
+#include "TWTronMessageSigner.h"
```

Comment:
Can this be a template for other target languages? For example, for Kotlin?
Then this file could be moved to a parent directory, let's say `codegen-v2/src/codegen/templates`

And also one line break 🙂

Reply:
This file is copied into `swift/Sources/Generated/` and only used for Swift, afaik. And the codegen should generate this file, anyway. Soon™

Reply:
See https://github.com/trustwallet/wallet-core/issues/3159